### ISSUE TITLE
Invalid @nodeAdded message format

### ISSUE THREAD

**Issue by cm0x4D** [cloudio-project/cloudio-endpoint-python#11]
In `@nodeAddded` messages the node contains a field `interfaces` that is not conform to the JSON format and causes the server to drop the message without processing. Only `implements` and `objects` fields are allowed as children of a node.

**Comment by cm0x4D** [comment]
### Exception when receiving the `@nodeAddded` message:

```
2018-01-25 11:00:22.156 ERROR 1 --- [TaskExecutor-15] c.h.c.c.m.l.AbstractLifecycleService     : Exception during @nodeAdded message handling:

com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "interfaces" (class ch.hevs.cloudio.cloud.model.Node), not marked as ignorable (2 known properties: "implements", "objects"])
```

**Comment by boozo-unlimited** [comment]
Fixed in 2ac3dab
